src/Redirect.js

```diff
@@ -0,0 +1,22 @@
+import RedirectException from './RedirectException';
+
+export default class Redirect {
+  constructor({ from, to }) {
```

Comment:
I think this reads a little bit better than using positional args. See examples/basic.